### DISCUSSION TITLE
Add new icon

### DISCUSSION THREAD

**Discussion by Helder-Machava** [devicons/devicon#1868] [Ideas]
Hi Everyone,
My name is Helder, I´m a design and also webdev, I'd like to contribute with adding a new icon of programing lang that is missing in the repo.
What can i do to help ?

**Comment by canaleal** [comment]
Hey @Helder-Machava welcome to the project! 

If you'd like to contribute please follow the steps outlined [here](https://github.com/devicons/devicon/blob/master/CONTRIBUTING.md#overview-on-submitting-icons).

If you have any questions about creating a pull request or creating a new icon, feel free to reach out on the [Discord Server](https://github.com/devicons/devicon/tree/master#discord-server) or simply create another discussion post.

**Reply by Snailedlt** [reply]
Thanks for helping out!
The README.md file is actually outdated, and will be removed in the future. The up to date version of those guidelines can be found in our wiki here -> https://github.com/devicons/devicon/wiki

**Reply by Helder-Machava** [reply]
Thank you!

**Comment by Helder-Machava** [comment]
I Finally got it! and i started to use it in my own profile 😀
Thanks for suporte, i hope to contribuite in adding much more icons missing...


![Untitled](https://github.com/devicons/devicon/assets/81919768/d5d94350-b1fc-4ff4-b20e-4e6f65df2572)

**Comment by Helder-Machava** [comment]
How long it takes to review a PR?

**Reply by Snailedlt** [reply]
It depends on how active the maintainers are. Atm it takes quite some time, because most of the maintainers aren't that active, which is why we are looking for new maintainers (see #1471)
